runtime/knative/Dockerfile.python38

```diff
@@ -1,32 +1,22 @@
-FROM python:3.8-slim-buster
```

Comment:
@aitorarjona @JosepSampe why only Python 3.8 need to be changed? Is this 3.8 specific?

Reply:
Python 3.8 Dockerfile was missing gunicorn in `RUN pip install ....` layer, so `CMD exec gunicorn --bind :$PORT --workers 1 --timeout 600 lithopsproxy:proxy` would fail when running a container with this image. I just copy-pasted the Python3.7 Dockerfile and changed `FROM python:3.7-slim-buster` to `FROM python:3.8.2-slim-buster`.